### DISCUSSION TITLE
Name clash found between static and dynamic tenant

### DISCUSSION THREAD

**Discussion by paultrigcode** [lorinkoz/django-pgschemas#127] [Q&A]
I am currently getting this error below
 ?: (pgschemas.W004) Name clash found between static and dynamic tenants: {'public'}"
 whenever  i run the command "python manage.py createsuperuser", Please any insights into why i am experiencing this. Thanks

**Comment by lorinkoz** [comment]
Hola @paultrigcode. Do you have a tenant in database whose `schema_name` is public?

**Reply by lorinkoz** [reply]
This could happen, for instance, when re-using a database from `django-tenants`.

**Reply by paultrigcode** [reply]
HI @lorinkoz  I have a tenant in my database whose schema is public, how can I fix the issue, I am trying to create superuser for each tenant including the public schema

**Reply by lorinkoz** [reply]
- You need to remove the entry for the 'public' schema in your tenant database.
- You need to add the user app both in the public and default sections of the TENANTS dict (see https://django-pgschemas.readthedocs.io/en/stable/basic.html#basic-configuration)

**Reply by paultrigcode** [reply]
I have done  that and i'm no longer getting the error, Thank you, I need to have a public schema , how can i have my public schema(tenant) setup so i can have a domain setup for it, because the entry point into the app lies on the public schema from which i can get to create other  tenants

**Reply by lorinkoz** [reply]
@paultrigcode You can read the docs here https://django-pgschemas.readthedocs.io/en/stable/basic.html#creating-tenants, this package has some differences in configuration as to django-tenants or django-tenant-schemas. If there's anything unclear in the documentation I'd be happy to fill the gap.

**Reply by paultrigcode** [reply]
I have gone through the documentation but i didnt see any documentation on how to configure superuser for the static tenants. I specified a Fall back domain "localhost" and i tried creating a superuser , but cant login into django admin using the fallback domain, Do you have any idea how to go about it. Thanks

**Reply by lorinkoz** [reply]
Hola @paultrigcode. Did you solve this?
